src/styles/blocks/page.scss

```diff
@@ -0,0 +1,14 @@
+.page {
+  height: 100%;
+  font-size: 30px;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.body {
```

Comment:
The class name `.body` is not following BEM naming convention. It seems like it is an element of `.page` block. Consider renaming it to `.page__body`.